src/InfoPanel/InfoPanel.jsx

```diff
@@ -0,0 +1,45 @@
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import "./InfoPanel.less";
+
+
+/**
+ * Base presentational component for the District Profile info panels.
```

Comment:
I'd change this comment given that the we'll now be using `InfoPanel`s for purposes other than the District Profile page